### DISCUSSION TITLE
No automatic redeploys after pushing to Github

### DISCUSSION THREAD

**Discussion by davidgbell** [vercel/vercel#6001] [Help]
There seems to be an issue regarding automatic redeploys after pushing project updates to Github. Previously Vercel would automatically redeploy the project picking up any changes, however it has suddenly stopped this behaviour. 

Is there any new updates that need to be configured for automatic redeploys?

**Comment by mcsdevv** [comment]
Hi @davidgbell, nothing has changed in a while on this. Have you checked the individual commit for validation errors that would prevent a Deployment being made?

Please check [this article](https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel) and if no luck, [contact support](vercel.com/support/request) with the commit SHA that you are unable to find the Deployment for.

**Reply by mrcm2000** [reply]
Read through the article and doubled checked everything and everything seemed to be up to date. Mine stopped deploying without any changes on my end as well on last commit today (previous commit was yesterday that did automatically deploy).

Submitted a ticket, but just dropping a note here that mine stopped deploying without any changes on my end.

**Reply by mrcm2000** [reply]
Update - just tried to create a new repo and deploy and got this error: 

`Unexpected response content-type (POST https://vercel.com/api/v6/projects?withUserCredentials=1): text/plain(502) An internal error occurred with Vercel. BAD_GATEWAY`

<img width="805" alt="Screen Shot 2021-03-22 at 12 16 04 PM" src="https://user-images.githubusercontent.com/905603/112045724-670f7e80-8b08-11eb-941b-0bcc212d7994.png">
